remote_vector_index_builder/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rchitale7/remote-index-build-service:faiss-base
```

Comment:
lets add a TODO on this so that we can replace the faiss base image with the actual image we need.